docs/run.md

```diff
@@ -28,10 +28,8 @@ _Optional_: Run full node on Mock DA:
 
 Full node RPC is accessible at `127.0.0.1:12346`
 
-To publish blocks on Mock DA, run these two on seperate terminals:
+To publish blocks on Mock DA, run the sequencer with `test_mode` in sequencer config set to false and run this script:
```

Comment:
If `test_mode` is set to false in the sequencer config, the sequencer will publish blocks every 2 seconds. To also publish mock DA blocks, run this script: